stdlib/public/core/Policy.swift

```diff
@@ -15,16 +15,49 @@
 //===----------------------------------------------------------------------===//
 // Standardized uninhabited type
 //===----------------------------------------------------------------------===//
-/// The return type of functions that do not return normally, that is, a type
-/// with no values.
+/// A type that has no values and can't be constructed.
 ///
-/// Use `Never` as the return type when declaring a closure, function, or
-/// method that unconditionally throws an error, traps, or otherwise does
-/// not terminate.
+/// Use `Never` as the return type of a function
+/// that doesn't ever return normally --- for example,
```

Comment:
I think eliding "ever" reads slightly better to me here, but I acknowledge the benefit of using an antonym of "never" to clarify the semantics here, especially to non-native English speakers.

```suggestion
/// that doesn't return normally --- for example,
```

Reply:
You're correct, I got to this wording like you said, by describing the behavior without repeating the word "never".  But your revision looks good.